internal/driver/fetch_test.go

```diff
@@ -382,6 +382,9 @@ func TestHttpsInsecure(t *testing.T) {
 		deadline := time.Now().Add(5 * time.Second)
 		for time.Now().Before(deadline) {
 			// Simulate a hotspot function.
+			for i := 0; i < 1e8; i++ {
```

Comment:
I have added a comment. The purpose is to ensure the samples mostly land onto the loop. As far as I can tell Go compiler doesn't elide empty loops now.

```
@aalexand0:tmp$ cat test.go
package main

func main() {
        for i := 0; i < 16; i++ {
        }
}
@aalexand0:tmp$ go build test.go
@aalexand0:tmp$ objdump -d test | grep -A10 "<main.main>:"
00000000004511c0 <main.main>:
  4511c0:       31 c0                   xor    %eax,%eax
  4511c2:       eb 03                   jmp    4511c7 <main.main+0x7>
  4511c4:       48 ff c0                inc    %rax
  4511c7:       48 83 f8 10             cmp    $0x10,%rax
  4511cb:       7c f7                   jl     4511c4 <main.main+0x4>
  4511cd:       c3                      retq   
  4511ce:       cc                      int3   
  4511cf:       cc                      int3   

00000000004511d0 <main.init>:
```